docs/metrics/metrics.md

```diff
@@ -0,0 +1,100 @@
+# Metrics
```

Comment:
@vigith This is still in Draft, but do you think this accomplishes what we discussed regarding providing a doc to explain the different metrics we can monitor on?

Reply:
how about we document the metrics as a table, eg [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics/blob/master/docs/ingress-metrics.md)